src/test/java/race/CarTest.java

```diff
@@ -0,0 +1,41 @@
+package race;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+
+    private Car car;
+    private RandomNumCondition randomNumCondition;
+
+    @BeforeEach
+    void setUp() {
+        randomNumCondition = new RandomNumCondition();
+        car = new Car();
+    }
+
+    @Test
+    @DisplayName("setDistance 테스트")
+    void addDistanceTest() {
+        //given
+        car.addDistance(new MoveCondition() {
+            @Override
+            public int getValue() {
+                return 1;
+            }
+
+            @Override
+            public boolean moveOrNot() {
+                return true;
+            }
+        });
```

Comment:
```suggestion
        car.addDistance(() -> true);
```
전략 객체의 메소드가 한 개일 경우 위와 같이 `람다식`으로 표현할 수 있답니다!